### ISSUE TITLE
onRegionDidChange not called on iOS

### ISSUE THREAD

**Issue by maxencehenneron** [nitaliano/react-native-mapbox-gl#1391]
I am trying to call onRegionDidChange but it never gets called.

```
<Mapbox.MapView
              styleURL={Mapbox.StyleURL.Street}
              centerCoordinate={this.state.coordinates}
              onRegionDidChange={this.onRegionDidChange}
              zoomLevel={11}
              ref={(c) => this._map = c}
              style={styles.map}
          > 
</Mapbox.MapView>
```

onRegionDidChange = () => console.log('region did change!')

It seems to be broken since 6.1.1. I downgraded to 6.1.0 and the event is firing correctly.

I'm using react native version 0.55.4

**Comment by ClayChristmas** [comment]
@maxencehenneron - I just tested this on 6.1.2 with an iPad simulator running react native version 0.54.4.  It fires for me without issues.  What are you doing that you expect the region to change?  I'm at zoom 3 and panning around or zooming in/out a bit seems to trigger this event.

**Comment by maxencehenneron** [comment]
I'll try to reproduce it with the example later this week. May be related to the native SDK version / react native version

**Comment by maxencehenneron** [comment]
Closed in favor of #1400
